docs/topics/forms/formsets.txt

```diff
@@ -126,6 +126,46 @@ affect validation.  If ``validate_max=True`` is passed to the
 :func:`~django.forms.formsets.formset_factory`, then ``max_num`` will affect
 validation.  See :ref:`validate_max`.
 
+.. _formsets-absolute-max:
+
+Limiting the maximum number of forms submitted
+----------------------------------------------
+
+.. versionadded:: 3.1
+
+The ``absolute_max`` parameter to
+:func:`~django.forms.formsets.formset_factory` provides the ability to limit
+the maximum number of forms that can be instantiated when supplying ``POST``
+data.  This is to protect against memory exhaustion attacks using forged POST
+requests::
+
+    >>> from django.forms.formsets import formset_factory
+    >>> from myapp.forms import ArticleForm
+    >>> ArticleFormSet = formset_factory(ArticleForm, absolute_max=10)
```

Comment:
This raises ``ValueError('absolute_max must be..`` because `DEFAULT_MAX_NUM = 1000`.